app/scripts/metamask-controller.test.js

```diff
@@ -109,6 +109,7 @@ describe('MetaMaskController', function () {
   const noop = () => undefined;
 
   before(async function () {
+    globalThis.AbortController = window.AbortController;
```

Comment:
As an aside, that I had to do this has me wondering when we'll fully migrate to `jest`.